docs/release-notes/8.12.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[release-notes-header-8.12.0]]
+== 8.12
+
+[discrete]
+[[release-notes-8.12.0]]
+=== 8.12.0
+
+[discrete]
+[[known-issue-8.12.0]]
+==== Known issues
+
+There are no known issues in 8.12.0.
+
+[discrete]
+[[breaking-changes-8.12.0]]
+==== Breaking changes
+
+There are no breaking changes in 8.12.0.
+
+[discrete]
+[[deprecations-8.12.0]]
+==== Deprecations
+
+There are no deprecations in 8.12.0.
+
+[discrete]
+[[features-8.12.0]]
+==== New features
+
+* Introduces the ability to assign alerts to specific users ({pull}170579[#170579], {pull}171589[#171589]).
+* Introduces Retrieval Augmented Generation (RAG) for Alerts, allowing you to give Elastic AI Assistant context about more alerts in your environment ({pull}172542[#172542]).
+* Enables alert suppression for threshold rules ({pull}171423[#171423]).
+* Adds an *Updates* tab to the prebuilt rules upgrade flyout to show differences between the installed and updated versions ({pull}172535[#172535], {pull}173187[#173187]).
+* Adds a setting that lets you exclude cold and frozen tiers from visual event analyzer queries ({pull}172162[#172162]).
+* Adds a tour to guide users through Timelines UI changes ({pull}172030[#172030]).
+* Adds a timeout option for Osquery queries, so you can customize the maximum time each query should run before timing out ({pull}169925[#169925]).
+* Introduces new grouping capabilities for CSPM and KSPM Findings data ({pull}169884[#169884]).
+* Adds the expandable alert details flyout to the rule preview panel ({pull}167902[#167902]).
+* Adds a button to rule execution error messages that lets you ask AI Assistant to diagnose errors ({pull}166778[#166778]).
```

Comment:
👍🏽

Reply:
Closing as this as been moved to enhancements sections 🕺🏼🕺🏼🕺🏼